deploy/templates/postgres-template.yml

```diff
@@ -0,0 +1,140 @@
+---
+kind: Template
+apiVersion: template.openshift.io/v1
+metadata:
+  name: migration-planner-db
+parameters:
+  - name: DB_SERVICE_NAME
+    description: The name of the OpenShift Service exposed for the database.
+    displayName: Database Service Name
+    value: migration-planner-postgres
+  - name: DB_SECRET_NAME
+    description: The name of the OpenShift Secret used for the database.
+    displayName: Database Secret Name
+    value: migration-planner-db
+  - name: DB_USER
+    description: Username for PostgreSQL user that will be used for accessing the database.
+    displayName: PostgreSQLUsername
+    value: admin
+  - name: DB_PASSWORD
+    description: Password for the PostgreSQL connection user.
+    displayName: PostgreSQL Password
+    value: adminpass
+  - name: DB_NAME
+    description: Name of the PostgreSQL database accessed.
+    displayName: PostgreSQL Database Name
+    value: planner
+  - name: POSTGRES_IMAGE
+    description: Postgres image (14 or latest).
+    displayName: Postgres Image
+    value: quay.io/sclorg/postgresql-12-c8s
+  - name: DB_VOLUME_CAPACITY
+    description: Volume space available for data, e.g. 512Mi, 2Gi.
+    displayName: Volume Capacity
+    value: 512Mi
+
+objects:
+  - kind: Service
+    apiVersion: v1
+    metadata:
+      name: ${DB_SERVICE_NAME}
+      labels:
+        app: ${DB_SERVICE_NAME}
+    spec:
+      ports:
+        - port: 5432
+          targetPort: 5432
+          protocol: TCP
+      selector:
+        app: ${DB_SERVICE_NAME}
+  - kind: Deployment
+    apiVersion: apps/v1
+    metadata:
+      name: ${DB_SERVICE_NAME}
+    spec:
+      selector:
+        matchLabels:
+          app: ${DB_SERVICE_NAME}
+      replicas: 1
+      template:
+        metadata:
+          labels:
+            app: ${DB_SERVICE_NAME}
+        spec:
+          containers:
+            - name: postgres
+              image: ${POSTGRES_IMAGE}
+              imagePullPolicy: IfNotPresent
+              ports:
+                - containerPort: 5432
+                  protocol: TCP
+              env:
```

Comment:
Can we here use just `envFromSecret` ? It would be less lines

Reply:
envFromSecret doesn't allow to change the environment variables names, this way we have control on the name

Reply:
Yep, but why do you need to change them? Isn't it OK to define the name in secret as you need it?

Reply:
This secret must have the same values like the secret provided by AppSRE in case of RDS, as this is used by the service.

Reply:
Ah, ok :)